peachjam/templates/peachjam/_citations_list.html

```diff
@@ -0,0 +1,33 @@
+{% load peachjam i18n %}
+{% for item in citations %}
+  {{ item.doc_type }} <span class="badge bg-secondary">{{ item.docs|length }}</span>
+  <ol>
+    {% if item.docs|length < 11 %}
+      {% for doc in item.docs|slice:":10" %}
+        <li>
+          <a href="{% url 'document_detail' frbr_uri=doc.work.frbr_uri|strip_first_character %}">{{ doc.title }}</a>
+        </li>
+      {% endfor %}
+    {% else %}
+      {% for doc in item.docs|slice:":10" %}
+        <li>
+          <a href="{% url 'document_detail' frbr_uri=doc.work.frbr_uri|strip_first_character %}">{{ doc.title }}</a>
+        </li>
+      {% endfor %}
```

Comment:
The two for loops are identical. You always want to show the first 10 items, no matter if there are less than 10 or more than 10. So the first ten can be shown, always, outside of an if statement. Then the exceptional case is when there are more than 10.